src/Avalonia.Visuals/Media/Brush.cs

```diff
@@ -18,13 +18,19 @@ public abstract class Brush : Animatable, IMutableBrush
         public static readonly StyledProperty<double> OpacityProperty =
             AvaloniaProperty.Register<Brush, double>(nameof(Opacity), 1.0);
 
+        /// <summary>
+        /// Defines the <see cref="Transform"/> property.
+        /// </summary>
+        public static readonly StyledProperty<Transform> TransformProperty =
```

Comment:
We have ITransform interface, which is used on Control (but not on Geometry).
Should we use it instead?

Reply:
I am converting type from Transform to ITransform as it needs to be converted to immutable for deffered renderer.

Reply:
I have changed to ITransform and introduced immutable transform to fix deferred rendering